### PR TITLE
CXTB-186- Can-Click-and-View-Facility-Device-Call-History-and-Access-Call-Recordings

### PR DESCRIPTION
**Pull request by ksantana-nts**
features/testing for CXTB-186

### CODE REVIEW

**Comment by ksantana-nts** [comment]
Evidence for test:

image

File: tests/page_objects/providers_care_platform_page.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ providers_user_dropdown_menu:
   user_shift_toggle_checkbox: //nav[@class='user-dropdown-menu']//label[contains(@class,'radio-toggle-button')]
   user_list_item_provider1: //div[contains(text(),'On Shift')]//following-sibling::div[not(contains(@class,'is-offline'))]//div[@class='users-dropdown-name'][contains(text(),'$AND_CLI_provider_user1_name$')][contains(.,'$AND_CLI_provider_user1_lastname$')][contains(.,'Provider')]
 
+
```
Comment (dguzmanneural) [inline tests/page_objects/providers_care_platform_page.yaml:36]: There's no need for this extra new line.

File: tests/page_objects/providers_care_platform_page.yaml [hunk 2]
```diff
@@ -68,6 +69,8 @@ providers_facilities_page:
   local_hospitals_button: //button[text()='Local Hospitals']
   devices_tab: //button[text()='Devices']
   title_subtab_local_hospital: //h4[text()='Local Hospitals & ED Contact']
+  call_history_accordion: (//div[@class='accordion box-scrollable-content pl -ml text-sm']/div[starts-with(@class, 'accordion-item')])
```
Comment (dguzmanneural) [inline tests/page_objects/providers_care_platform_page.yaml:72]: Could you use only the parts of the class that matters? all that part of the class ("pl -ml text-sm") increase the probabilities of getting this selector to be obsolete. Also, you don't need to wrap all inside of parenthesis if you are not going to use an index (Ex: [1]) or function(Ex: [last()]) that needs to work with the rest of the selector as a whole.

Comment (ksantana-nts) [reply tests/page_objects/providers_care_platform_page.yaml:72]: @dguzmanneural is there perhaps another way recommended to find the element? Without that part of the class, the element is not located in the test and it doesn't have many unique attributes. 

image

File: tests/cases/providers_care_platform/case_providers_facilities.yaml [hunk 3]
```diff
@@ -101,3 +101,59 @@ TestProviderCanClickAndViewFacilitiesDevice:
...
+    # Click to open first option Accordion sub-tab for Call History
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.providers_facilities_page.call_history_accordion$
+    # Click on Telehealth request form
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.providers_facilities_page.telehealth_request_form$
+    - Type: sleep
+      Time: 3
```
Comment (dguzmanneural) [inline tests/cases/providers_care_platform/case_providers_facilities.yaml:159]: Why would you have a sleep action at the end?

**Review by csanchez-nts — CHANGES_REQUESTED** [review]

**Pull request closed without merging**